root/src/main/java/com/example/servingwebcontent/MovieListController.java

```diff
@@ -55,6 +56,32 @@ public String test() throws java.io.FileNotFoundException {
 
 		connection.closeConnection();
 
+		return "test";
+	}**/
+
+	@GetMapping(value = "/test/{movieId}")
+	public String test (@PathVariable String movieId) throws java.io.FileNotFoundException {
+		BufferedReader reader;
+		String user = "", pass = "";
+	
+		try {
+			reader = new BufferedReader (new FileReader("./userInfo.txt"));
+			String line = reader.readLine();
```

Comment:
you can combine this line and the next line and the following 2 lines. You don't need the "line" variable.